UI/src/view/dynamicAssetForm.js

```diff
@@ -655,6 +655,13 @@
       var rootElement = $('#feature-attributes');
       _assetTypeConfiguration = assetTypeConfiguration;
 
+      var updateStatusForMassButton = function(element) {
+        if(assetTypeConfiguration.selectedLinearAsset.isSplitOrSeparated()) {
+          element.prop('disabled', !(me.isSaveable() && me.isSplitOrSeparatedAllowed()));
+        } else
+          element.prop('disabled', !(me.isSaveable()));
+      };
+
```

Comment:
In **else** condition don't we need to user the dirty information ? like it is done on 
`            var updateStatus = function(element) {
                if(assetTypeConfiguration.selectedLinearAsset.isSplitOrSeparated()) {
                    element.prop('disabled', !(me.isSaveable() && me.isSplitOrSeparatedAllowed()));
                } else
                    element.prop('disabled', !(me.isSaveable() && assetTypeConfiguration.selectedLinearAsset.isDirty()));
            };` 

**Note** if it was needed we can reuse the same method

Reply:
Tallenna should be enabled when both On/Off is checked. If switching to On and then Off, it won't be enabled with that condition.